### DISCUSSION TITLE
onthisday/05/05-01/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#17] [General]
# onthisday/05/05-01/

Details for On This Day are genarated by the posts added to the website so the content is subject to changes/updates over time.

https://fanzoflenazavaroni.github.io/onthisday/05/05-01/

**Comment by StephenMolloy1** [comment]
I found this Getty photograph of Lucille Ball at the Share party dated 1st May 1974. As we know, these dates aren't always accurate so it would be nice to find something else with the same date just to confirm it.

https://www.gettyimages.co.uk/photos/lucille-ball-may-1974?assettype=image&editorialproducts=archival&family=editorial&phrase=Lucille%20Ball%20May%201974&sort=mostpopular

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the photo, I have a page for the show but it was not showing up in the OnThisDay as I had no date for it, while we can't be sure of the date as I agree with you on the inaccurate of photo sharing websites I have now added the 1st so that the page will now show up for visitors to the OnThisDay section. I updated the page with the photo you shared with me and added a link to the website for SHARE it has a number of clips from various shows but sadly none of Lena.
